app/scripts/modules/core/delivery/executionGroup/execution/ExecutionMarker.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from 'react';
 import * as ReactGA from 'react-ga';
 
-import { IExecutionStageSummary } from 'core/domain/IExecutionStage';
+import { IExecution, IExecutionStageSummary } from 'core/domain';
 import { OrchestratedItemRunningTime } from './OrchestratedItemRunningTime';
-import { Tooltip } from 'core/presentation/Tooltip';
 import { duration } from 'core/utils/timeFormatters';
 
```

Comment:
I'm good with the spaces and have updated my IDE to include them in import statements.